### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by cp-ishita-g**
- [x] Edit place screen make it scrollable in landscape mode - currently it is stuck when there is landscape mode
- [x] Check all screen in landscape mode - and make change in required screen
- [x] Choose place name screen make change to add place as primary button
- [x] Setting screen -> edit space -> remove border from toggle switch
- [x] Setting screen -> edit space -> change color of uneditable switch of other user
- [x] Setting screen -> edit space -> change background color of user profile image when there is no profile
- [x] Fix place added popup ui overflow in landscape mode

Simulator Screenshot - iPhone 15 Pro - 2024-08-20 at 15 10 26  Simulator Screenshot - iPhone 15 Pro - 2024-08-20 at 15 10 40  Simulator Screen Recording - iPhone 15 Pro - 2024-08-20 at 16 50 48

### CODE REVIEW

**Comment by kaushiksaliya** [comment]
This edit space related changes was already done, you can check [#74](https://github.com/canopas/yourspace-flutter/pull/74) mr.

**Comment by cp-ishita-g** [comment]
> This edit space related changes was already done, you can check [#74](https://github.com/canopas/yourspace-flutter/pull/74) mr.

i already sync that

**Comment by kaushiksaliya** [comment]
fix this add place floatingActionButton padding.

Screenshot from 2024-08-21 09-43-28

**Comment by kaushiksaliya** [comment]
> > This edit space related changes was already done, you can check [#74](https://github.com/canopas/yourspace-flutter/pull/74) mr.
> 
> i already sync that

then change your mr description.

**Comment by kaushiksaliya** [comment]
revert this user profile changes, this is not visible

Screenshot from 2024-08-21 09-58-56

File: app/lib/ui/flow/setting/space/edit_space_screen.dart [hunk 1]
```diff
@@ -211,9 +213,10 @@ class _EditSpaceScreenState extends ConsumerState<EditSpaceScreen> {
             }
           },
           activeTrackColor: context.colorScheme.primary,
-          inactiveTrackColor: context.colorScheme.containerHigh,
-          inactiveThumbColor: context.colorScheme.onPrimary,
-          trackOutlineColor: WidgetStateProperty.all(Colors.transparent),
+          inactiveTrackColor: context.colorScheme.outline,
+          inactiveThumbColor: context.colorScheme.textPrimaryDark,
+          trackOutlineColor: WidgetStatePropertyAll(context.colorScheme.outline),
```
Comment (kaushiksaliya) [inline app/lib/ui/flow/setting/space/edit_space_screen.dart:218]: revert this no need to show border in switch

Comment (cp-ishita-g) [reply app/lib/ui/flow/setting/space/edit_space_screen.dart:218]: we required border for dark theme, we need to keep it, i will make this same change in edit space screen also

**Comment by kaushiksaliya** [comment]
fix this switch toggle issue.

[Fix switch.webm](https://github.com/user-attachments/assets/ec2898b6-347a-4b81-b461-89b14107dac0)

File: app/lib/ui/flow/geofence/edit/edit_place_screen.dart [hunk 2]
```diff
@@ -99,46 +99,59 @@ class _EditPlaceViewState extends ConsumerState<EditPlaceScreen> {
...
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            AspectRatio(
-              aspectRatio: 1.85,
-              child: _googleMapView(
-                place.latitude,
-                place.longitude,
-                state.isAdmin,
-                place.radius,
+    return OrientationBuilder(
```
Comment (kaushiksaliya) [inline app/lib/ui/flow/geofence/edit/edit_place_screen.dart:110]: why we need to wrap OrientationBuilder?

File: app/lib/ui/flow/geofence/edit/edit_place_screen.dart [hunk 3]
```diff
@@ -405,32 +418,23 @@ class _EditPlaceViewState extends ConsumerState<EditPlaceScreen> {
...
+                  foreground: context.colorScheme.alert,
+                  onPressed: notifier.onTapDeletePlaceBtn,
+                )
+              : Text(
+                  context.l10n.edit_place_only_admin_edit_text,
+                  style: AppTextStyle.body2
+                      .copyWith(color: context.colorScheme.textSecondary),
+                  textAlign: TextAlign.center,
+                )
+        ],
```
Comment (kaushiksaliya) [inline app/lib/ui/flow/geofence/edit/edit_place_screen.dart:439]: add horizontal and bottom padding.

**Comment by cp-ishita-g** [comment]
> fix this switch toggle issue.
> 
>  Fix.switch.webm

this issue arise in this https://github.com/canopas/yourspace-flutter/pull/74 mr which changes was done by you

File: app/lib/ui/flow/geofence/add/addnew/add_new_place_screen.dart [hunk 4]
```diff
@@ -43,40 +43,42 @@ class _AddNewPlaceViewState extends ConsumerState<AddNewPlaceScreen> {
...
-            _searchTextField(),
-            const SizedBox(height: 56),
-            _locateOnMapView(),
-            const SizedBox(height: 40),
-            if (state.places.isNotEmpty || state.loading) ...[
-              Text(
-                context.l10n.add_new_place_suggestion_text,
-                style: AppTextStyle.caption.copyWith(
-                  color: context.colorScheme.textDisabled,
+    return SafeArea(
```
Comment (kaushiksaliya) [inline app/lib/ui/flow/geofence/add/addnew/add_new_place_screen.dart:46]: why we required SafeArea?

File: app/lib/ui/flow/geofence/edit/edit_place_screen.dart (hunk 2 above)
Comment (cp-ishita-g) [reply app/lib/ui/flow/geofence/edit/edit_place_screen.dart:110]: to manage ui for landscape mode

File: app/lib/ui/flow/geofence/add/addnew/add_new_place_screen.dart (hunk 4 above)
Comment (cp-ishita-g) [reply app/lib/ui/flow/geofence/add/addnew/add_new_place_screen.dart:46]: to manage ui for landscape mode

**Comment by kaushiksaliya** [comment]
fix this add place dialog alignment it should be centre.

Screenshot from 2024-08-21 10-47-28

**Comment by kaushiksaliya** [comment]
fix button overflow in landscape mode.

Screenshot from 2024-08-21 11-00-30

**Review by kaushiksaliya — APPROVED** [review]

**Pull request merged**